tasks/math/calculator/index.test.ts

```diff
@@ -0,0 +1,19 @@
+import { calculator } from "./index";
+
+describe("calculator", () => {
+  it("should return 4", () => {
+    expect(calculator(2, "+", 2)).toBe(4);
+  });
+  it("should return 4", () => {
+    expect(calculator(2, "*", 2)).toBe(4);
+  });
+  it("should return 2", () => {
+    expect(calculator(4, "/", 2)).toBe(2);
+  });
+  it("should return 3", () => {
+    expect(calculator(7, "-", 4)).toBe(3);
+  });
+  it("should return error", () => {
+    expect(calculator).toThrowError(new Error("Calculator does not support the provided operator"));
```

Comment:
You should hace call statement to test this case properly
![image](https://user-images.githubusercontent.com/95697276/182117359-61e99397-934e-48f3-8eb5-284380727bf3.png)